cmd/eib/main.go

```diff
@@ -92,12 +94,12 @@ func validateImageConfigDir(configDir string) error {
 func main() {
 	imageConfig, buildConfig, err := processArgs()
 	if err != nil {
-		log.Error(err)
+		zap.L().Error("parsing CLI arguments", zap.Error(err))
```

Comment:
```suggestion
		zap.L().Fatal("parsing CLI arguments", zap.Error(err))
```

We need to either log on `ERROR` level and manually `os.Exit(1)` here or use `Fatal()` which will log on `FATAL` level and exit automatically. We can't proceed with invalid config.